src/doc/rustc/src/platform-support.md

```diff
@@ -112,6 +114,8 @@ The `std` column in the table below has the following meanings:
 
 [`no_std`]: https://rust-embedded.github.io/book/intro/no-std.html
 
+**NOTE:** Tier 2 targets, currently, do not have access to the `rust-docs` component, and as such do not build documentation.
```

Comment:
This has caused and effect backwards: tier 2 targets don't have docs available *because* we choose not to built it.
```suggestion
**NOTE:** Tier 2 targets currently do not build the `rust-docs` component.
```